From 607d529c70e8f021003237a674beedec7b422de5 Mon Sep 17 00:00:00 2001
From: Sharath Srikanth Chellappa <sharathsr@microsoft.com>
Date: Mon, 7 Jul 2025 12:43:42 -0700
Subject: [PATCH] Modifying Golang version and adding makefile targets for
 development

These changes will potentially be removed as the upstream carries out a new release.

1. Go version upgraded from 1.23.x to 1.24.0 in:
    - Dockerfile
    - go.mod
    - hack/tools/go.mod

---
 go.mod            | 2 +-
 hack/tools/go.mod | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/go.mod b/go.mod
index a312010..d59a315 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module sigs.k8s.io/cluster-api-provider-kubevirt
 
-go 1.23.6
+go 1.24.0
 
 require (
 	github.com/go-logr/logr v1.4.2
diff --git a/hack/tools/go.mod b/hack/tools/go.mod
index cf216bb..7e19388 100644
--- a/hack/tools/go.mod
+++ b/hack/tools/go.mod
@@ -1,6 +1,6 @@
 module sigs.k8s.io/cluster-api-provider-aws/hack/tools
 
-go 1.23.0
+go 1.24.0
 
 require (
 	gotest.tools/gotestsum v1.12.0
-- 
2.49.0

